docker-compose.yaml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+
+services:
+  mongodb:
+    image: bitnami/mongodb:4.4.15
+    platform: linux/amd64
+    environment:
+      MONGODB_USERNAME: pritunl
+      MONGODB_DATABASE: pritunl
+      MONGODB_PASSWORD: pritunl
```

Comment:
WDYT on moving creds to a separate `.env` file, to keep it unknown during `cat docker-compose.yaml` ?

Reply:
do you mean variable substitutions in compose file or using `env_file` for each service?

Reply:
I mean `env_file`